### PR TITLE
Add pytreeLidarprofileJsonUrl for desktop interface

### PR DESCRIPTION
**Pull request by llienher**

### CODE REVIEW

**Review by sbrunner — APPROVED** [review]

File: geoportal/vars.yaml [hunk 1]
```diff
@@ -139,6 +139,7 @@ vars:
         #   - https://localhost:3001/webcomponents/index.ts
 
         sitnFeedbackPath: custom/feedback
+        pytreeLidarprofileJsonUrl: https://sitn.ne.ch/pytree
```
Comment (sbrunner) [inline geoportal/vars.yaml:142]: Also, should be removed from desktop_alt

**Review by sbrunner — APPROVED** [review]

File: geoportal/vars.yaml (hunk 1 above)
Comment (llienher) [reply geoportal/vars.yaml:142]: So in default and not in both ?

Comment (sbrunner) [reply geoportal/vars.yaml:142]: desktop_alt extends desktop, and we don't need it on mobile, then only here :-)

Comment (llienher) [reply geoportal/vars.yaml:142]: Done.

**Pull request merged**